frontend/packages/topology/src/components/graph-view/components/nodes/decorators/Decorator.tsx

```diff
@@ -79,6 +79,7 @@ const Decorator: React.FunctionComponent<DecoratorTypes> = ({
         }}
         role="button"
         aria-label={ariaLabel}
+        data-test={`decorator-${ariaLabel}`}
```

Comment:
Couldn't we check for the aria-label instead? As a benefit of this, we also test that our a11y doesn't break in the future and don't need to apply the same information twice?

If you want to keep this `data-test`, I think we should use a space as a separator to allow lookups via `~=` as well.

```suggestion
```

Reply:
Updated it without the hyphen